src/nvim/buffer_defs.h

```diff
@@ -1154,6 +1172,9 @@ struct window_S {
   int w_tagstackidx;                    /* idx just below active entry */
   int w_tagstacklen;                    /* number of tags on stack */
 
+  int w_floating;                       /* if the window is floating */
+  FloatMode w_float_mode;
```

Comment:
You can’t hold `kFloatAnchorNW | kFloatStandalone` in `FloatMode`, need generic integer type.